gcovr/json_generator.py

```diff
@@ -71,6 +72,20 @@ def gcovr_json_files_to_coverage(filenames, covdata, options):
             file_path = os.path.join(
                 os.path.abspath(options.root),
                 os.path.normpath(gcovr_file['file']))
+
+            # Return if the filename does not match the filter
+            # Return if the filename matches the exclude pattern
```

Comment:
Comment should be moved to if clause and Return can be replaced with ignore.

Reply:
Thanks! It was copy-paste error from `gcov.py` file :).